apps/client/src/app/modules/list/data/ListRowSerializationHelper.ts

```diff
@@ -0,0 +1,315 @@
+import { GarlandToolsService } from '../../../core/api/garland-tools.service';
+import { LocalizedDataService } from '../../../core/data/localized-data.service';
+import { I18nToolsService } from '../../../core/tools/i18n-tools.service';
+import { I18nName } from '../../../model/common/i18n-name';
+import { Ingredient } from '../../../model/garland-tools/ingredient';
+import { MobNamePipe } from '../../../pipes/pipes/mob-name.pipe';
+import { VenturesComponent } from '../../item-details/ventures/ventures.component';
+import { DataType } from '../data/data-type';
+import { CraftedBy } from '../model/crafted-by';
+import { GatheredBy } from '../model/gathered-by';
+import { Instance } from '../model/instance';
+import { getItemSource, ListRow } from '../model/list-row';
+import { StoredNode } from '../model/stored-node';
+import { Vendor } from '../model/vendor';
+
+interface NameHolder { name: string, itemIdName?: string }
+interface ItemNameHolder { id: number | string, itemId?: string, name?: string, itemIdName?: string };
+interface NPCNameHolder { id: number, npcId?: number, npcName?: string, zoneName?: string };
+interface ZoneInfo { id?: number; npcId?: number; npcName?: string; zoneName?: string; zoneId?: any; zoneid?: any; mapId?: any; };
+interface MapInfo { zoneId?: any; zoneid?: any; mapId?: any; };
+interface CurrencyInfo { currencies: any[]; items: any[]; };
+interface NpcTradeData { npcs: any; trades: CurrencyInfo[]; };
+interface MonsterInfo { id: any; itemId?: any; position?: any; npcId?: number; npcName?: string; zoneName?: string; zoneId?: any; zoneid?: any; mapId?: any; };
+interface MonsterInfoWithMobName extends MonsterInfo, NameHolder { }
+interface VendorWithName extends Vendor, NameHolder { }
+interface InstanceWithName extends Instance, NameHolder { }
+interface GatherWithName extends GatheredBy, NameHolder { }
+
+export class ListRowSerializationHelper {
+
+  public static getData<T = any>(row: ListRow, type: DataType, isObject = false): T {
+    return getItemSource<T>(row, type, isObject);
+  }
```

Comment:
What's the purpose of this method? You can also just use `getItemSource` directly.